client/src/main/java/org/apache/rocketmq/client/Validators.java

```diff
@@ -107,4 +110,19 @@ public static void isNotAllowedSendTopic(String topic) throws MQClientException
         }
     }
 
+    public static void checkTopicConfig(final TopicConfig topicConfig) throws MQClientException {
+        if (!PermName.isValid(topicConfig.getPerm())) {
+            throw new MQClientException(ResponseCode.NO_PERMISSION,
+                String.format("brokerPermission value: %s is invalid.", topicConfig.getPerm()));
```

Comment:
`brokerPermission value` should be `topicPermisson value`, and  is more accurate